### ISSUE TITLE
HyperRipper overall progress

### ISSUE THREAD

**Issue by raptcha** [elbereth/DragonUnPACKer#167]
I'm using HR to find audio files in a 5,000,000 KB PAK file, and it's been almost a week. It would be nice if there was some indication of how much time there is left. 
Actually this might be an issue report. It finds exactly 15 items, the progress bar fills, and it starts over again. It says it has found nearly 40,000 items so far, but as far as I can tell, it's finding the same 15 items over and over again (at the end of each cycle, it shows the same 5). I don't know how many sound files are actually in the pak, but I know it's at least a few hundred.

**Comment by raptcha** [comment]
update, I gave up and ended the process, then I opened up the HRF. sure enough, about a zillion copies of the same files.

**Comment by raptcha** [comment]
update, I gave up and ended the process, then I opened up the HRF. sure enough, about a zillion copies of the same files.

**Comment by elbereth** [comment]
Yes, it is obviously some bug in the logic. It should not take more than a couple hours even for the biggest file on earth and the slowest computer...

But I need you to give me more info :
- What version of Dragon UnPACKer are you using ? If not the latest (5.7.1 Nightly-g60e29a6) [try it](https://www.elberethzone.net/en/dragon-unpacker-dev.html#download-nightly) and let me know.
- What were the settings in HyperRipper (formats enabled, buffer size, rollback size, etc...) ?
- If I understand correctly, the file was more than 2GB, around 5GB ? Was it from a game ? If so, which one. I got an extensive game library, maybe I have the file and can test it myself.

**Comment by raptcha** [comment]
5.7.0, i'll try the latest next.
all the audio formats are checked, except 669, 669 ext, and IT. 
i tried again several times and the search speed was significantly faster, but still finding the same 15 over and over. I tried searching for only OGG, and it found 14 and stopped. 
buffer size 128KB, rollback size 16B
yes, around 5GB. It's from the game We Happy Few.

**Comment by raptcha** [comment]
I tried the latest version. This time it found the same 15 items, then found about 12,000 wav files. none of the ones that I tried would play. The OGG files play fine.
